binding/Binding/GRContext.cs

```diff
@@ -99,36 +79,24 @@ public void GetResourceCacheUsage (out int maxResources, out long maxResourceByt
 			}
 			maxResourceBytes = (long)maxResBytes;
 		}
-		
-		public void ResetContext (GRGlBackendState state)
-		{
-			ResetContext ((uint) state);
-		}
 
-		public void ResetContext (GRBackendState state = GRBackendState.All)
-		{
-			ResetContext ((uint) state);
-		}
+		public void ResetContext (GRGlBackendState state) =>
+			ResetContext ((uint)state);
 
-		public void ResetContext (uint state)
-		{
+		public void ResetContext (GRBackendState state = GRBackendState.All) =>
+			ResetContext ((uint)state);
+
+		public void ResetContext (uint state) =>
 			SkiaApi.gr_context_reset_context (Handle, state);
-		}
 
-		public void Flush ()
-		{
+		public void Flush () =>
 			SkiaApi.gr_context_flush (Handle);
-		}
 
-		public int GetMaxSurfaceSampleCount (SKColorType colorType)
-		{
-			return SkiaApi.gr_context_get_max_surface_sample_count_for_color_type (Handle, colorType);
-		}
+		public int GetMaxSurfaceSampleCount (SKColorType colorType) =>
+			SkiaApi.gr_context_get_max_surface_sample_count_for_color_type (Handle, colorType);
 
+		[EditorBrowsable (EditorBrowsableState.Never)]
 		[Obsolete ("Use GetMaxSurfaceSampleCount(SKColorType) instead.")]
-		public int GetRecommendedSampleCount (GRPixelConfig config, float dpi)
-		{
-			return GetMaxSurfaceSampleCount (config.ToColorType ());
-		}
+		public int GetRecommendedSampleCount (GRPixelConfig config, float dpi) => 0;
```

Comment:
This was incorrectly selecting the max, which often resulted in a too value. Originally it was the "minimum" value that was "guessed" from properties. The max is actually the highest supported based on properties - which is usually higher that what the surface was actually created with. This causes some issues on some platforms.